pkgs/top-level/all-packages.nix

```
@@ -26582,7 +26582,7 @@ in
 
   utox = callPackage ../applications/networking/instant-messengers/utox { };
 
-  valentina = libsForQt514.callPackage ../applications/misc/valentina { };
+  valentina = libsForQt512.callPackage ../applications/misc/valentina { };
```

In anycase, if the PR will be merged as is, please add:

```suggestion
  # Version 0.8.x or above should support qt5.15
  valentina = libsForQt512.callPackage ../applications/misc/valentina { };
```